object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -75,4 +92,11 @@ You can define a private Object Store that is not defined as a global element an
 </idempotent-message-validator>
 ----
 
-The example provides the idempotent message validator with a custom store that  only it can access.
+The example provides the idempotent message validator with a custom store that only it can access.
```

Comment:
```suggestion
The example provides the idempotent message validator with a custom store that only the validator can access.
```

Reply:
It wasn't immediately clear whether "it" referred to the validator or the custom store.